curriculum/challenges/english/25-front-end-development/lecture-working-with-loops-and-sequences/6839e4d62ae09d1661e4cf9b.md

```diff
@@ -1,14 +1,115 @@
 ---
 id: 6839e4d62ae09d1661e4cf9b
 title: What Are List Comprehensions and What Are Some Useful Functions to Work With Lists?
-challengeType: 11
-videoId: nVAaxZ34khk
+challengeType: 19
 dashedName: what-are-list-comprehensions-and-what-are-some-useful-functions-to-work-with-lists
 ---
 
-# --description--
+# --transcript--
 
-Watch the video or read the transcript and answer the questions below.
+For the past few lectures, you have been getting comfortable working with loops like this:
+
+```py
+even_numbers = []
+
+for num in range(21):
+    if num % 2 == 0:
+        even_numbers.append(num)
+
+print(even_numbers)
+```
+
+This example creates a new empty list called `even_numbers` and loops through a sequence of numbers between `0` and `20`. Inside the loop, there's a condition that checks if the current number has a remainder of `0` when divided by `2`. This is used to determine if the number is even. If the condition is `True`, then the current `num` is appended at the end of the `even_numbers` list. Finally, we print the `even_numbers` list to the console.
+
+While this code works, there is a more concise way to write this that uses list comprehension instead. List comprehension allows you to create a new list in a single line by combining a loop and condition directly within square brackets or by passing them to the `list()` function. This makes the code shorter and often easier to read.
+
+Here is the refactored example from earlier using square brackets:
+
+```py
+even_numbers = [num for num in range(21) if num % 2 == 0]
+print(even_numbers)
+```
+
+And here it is using the `list()` function:
+
+```py
+even_numbers = list(num for num in range(21) if num % 2 == 0)
+print(even_numbers)
+```
```

Comment:
Technically this is a generator expression passed to the list constructor. I'm not sure if we want to keep this. If we remove it, there is a sentence to change in the previous paragraph (" or by passing them to the `list()` function").